adagucserverEC/adagucserver.cpp

```diff
@@ -252,6 +252,7 @@ int main(int argc, const char *argv[]){
         setDebugFunction(serverDebugFunction);
         
         CT::string fileInfo = CGetFileInfo::getLayersForFile(file.c_str());
+        CDFObjectStore::getCDFObjectStore()->clear();
```

Comment:
Ik snap niet zo goed waarom we dit hier doen, is dit ook een bugfix die op de KDC respository is gedaan?